src/main/scala/net/liftmodules/ng/Angular.scala

```diff
@@ -353,12 +353,12 @@ object Angular extends DispatchSnippet with AngularProperties with LiftNgJsHelpe
   /**
    * Transparently wraps `NgModel => Box[Any]` functions appropriately for `defModelToAny`/`jsonCall`
    */
-  implicit def wrapModelFnBox[M](f: M => Box[Any]) = ModelFnBox(f)
+  implicit def wrapModelFnBox[M](f: M => Box[Any]): ModelFnBox[M] = ModelFnBox(f)
```

Comment:
@joescii your suggestion about removing this wrappers. Should we keep them around as deprecated artifacts, and maybe remove them in a subsequent version?

Reply:
They can be removed entirely. They're an artifact of enforcing the `NgModel` constraint at compile time. Without the wrappers, the code would compile but fail at runtime. I testing removing these, and everything seems fine.

Reply:
plus one for version numbering change, this sounds a significant and worthwhile change.